.github/workflows/release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Build package
         run: |
-          uv pip install build twine wheel
+          uv pip install --system build twine wheel
```

Comment:
```
Build & publish package to pypi
Using latest version of uv because no version is provided
```
-- https://github.com/crate/crate-python/actions/runs/11703279293